webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -833,6 +833,10 @@ protected function validateLanguage(Event $event, EventData $data): void
             ]);
         } else {
             $this->removeWarning($event->type, $data->id, ExternalSourceWarning::TYPE_DATA_MISMATCH);
+
+            $toCheck = ['extensions' => $data->extensions];
```

Comment:
Aren't the extensions standardized?

Reply:
No, it is fine to take them from the primary. We should just make sure we publish the right extensions in any docs.

Reply:
And that the CCS'es use said extensions 🫠